### PR TITLE
ci: Get tests running on both windows and linux

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

**Comment by jan-auer** [comment]
@Swatinem slightly updated the title to conform with the commit guidelines.

File: tests/test_modulefinder.c [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 SENTRY_TEST(test_module_finder)
 {
+// the module finder is not yet implemented on linux
+#if defined(SENTRY_PLATFORM_DARWIN) || defined(SENTRY_PLATFORM_WINDOWS)
```
Comment (jan-auer) [inline tests/test_modulefinder.c]: Maybe add a TODO comment here, so we can track this?

Comment (jan-auer) [reply tests/test_modulefinder.c]: Also, let's confirm, but iirc this is required for Android.

**Comment by Swatinem** [comment]
changed the comment and created a task to get a module finder on linux going.

**Pull request merged**